rcl/include/rcl/logging_external_interface.h

```diff
@@ -31,7 +31,7 @@
 RCL_PUBLIC
 RCL_WARN_UNUSED
 rcl_ret_t
-rcl_logging_external_initialize(const char * config_file);
+rcl_logging_external_initialize(const char * config_file, rcutils_allocator_t allocator);
```

Comment:
For semantics shouldn't this be an rcl_allocator_t? 

That's what's used internallly currently: https://github.com/ros2/rcl/blob/eabe42686688b652be0aa020db2b396c393536cc/rcl/src/rcl/logging.c#L41

Of course it's the same via typedef: https://github.com/ros2/rcl/blob/3b9475bb142bc6226b7441e5926eecb3f3dbf9d9/rcl/include/rcl/allocator.h#L25

Reply:
The problem is that these packages shouldn't depend on `rcl`, as it creates a circular dependency.  Also, all of the functions I'm going to eventually call are in `rcutils`, so I think it makes more sense as an `rcutils_allocator_t`.

Reply:
👍 That actually is a good idea. Though they are just a `typedef` to one another right now.